src/index.js

```diff
@@ -11,7 +11,8 @@ const habitat = Widget => {
       selector = null,
       inline = false,
       clean = false,
-      clientSpecified = false
+      clientSpecified = false,
+      defaultProps
```

Comment:
1 final thing to keep the code as consistent as possible <3

should we have the defaultProps default value in here like other options?

I mean `defaultProps = {}` so it is consistent with other options :)

if we default it here we can remove the default from lib.js> preactRender(..)

apologise for not noticing this earlier